chemreg/substance/tests/test_views.py

```diff
@@ -73,3 +78,32 @@ def test_source_patch(client, admin_user, source_factory):
     )
     assert resp.status_code == 200
     assert resp.data["name"] == "a-new-name"
+
+
+def test_substance_type_view():
+    """Tests that the Substance Type View Set, is a sub class of the Model View Set."""
+    assert issubclass(SubstanceTypeViewSet, ModelViewSet)
+
+
+@pytest.mark.django_db
+def test_substance_type_post(client, admin_user, substance_type_factory):
+    client.force_authenticate(user=admin_user)
+    stf = substance_type_factory.build()
+    resp = client.post(
+        "/substanceTypes",
+        {"data": {"type": "substanceType", "attributes": stf.initial_data}},
+    )
+    assert resp.status_code == 201
+
+
+@pytest.mark.django_db
+def test_substance_type_get(client, admin_user, substance_type_factory):
+    client.force_authenticate(user=admin_user)
+    substance_type_factory.create()
+    resp = client.get("/substanceTypes")
+    assert resp.status_code == 200
+    for result in resp.data["results"]:
```

Comment:
```suggestion
    assert resp.data["results"]
    for result in resp.data["results"]:
```
this is very minor, but when you had used the `build` in this test before all of the assertions below weren't being run because there was no list of results, so we should assert that there is a list of results. If the list is empty the assertion added will fail.